src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs

```diff
@@ -458,6 +458,7 @@ public void FromSourceBatch_NullSources_ThrowsArgumentNullException()
         [InlineData(null)]
         [InlineData("")]
         [InlineData("cmdArgs")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Looks no issue in test or code, somehow occasionally failing in .Net framework, most likely caused by CI")]
```

Comment:
Is it possible to get a bit more info on why is the test failing? Seems like the message here is a bit vague.

Reply:
"Bug in .NET Framework"

Reply:
Changed into: `Occasionally fails in .NET framework, probably caused from a very edge case bug in .NET Framework which we will not be fixing`

Reply:
I'm not sure about the part "which we will not be fixing" as it leaves very little room for future change of plans, but I won't block this PR for that.

Reply:
For me `which we will not be fixing` is the main reason for skipping the test, but I can delete that if needed